tests/test_data/write_specified_cartesian_speaker.accepted.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@tomjnixon does this file now show the output you would expect? Particularly the bound="min" etc.. bits and the screenEdgeLock?